1-mandatory/1-writers.js

```diff
@@ -72,7 +78,11 @@ Exercise 3:
 */
 
 function logAliveWritersInTheirForties() {
-  // write your code here
+  const aliveOlderWriters = writers.filter((eachWriter) => eachWriter.age > 39 && eachWriter.age < 50 && eachWriter.alive === true);
+  const writerName = aliveOlderWriters.map((eachWriter) => eachWriter.firstName);
+  const writerSurname = aliveOlderWriters.map((eachWriter) => eachWriter.lastName);
+  const writerAge = aliveOlderWriters.map((eachWriter) => eachWriter.age);
+  console.log(`Hi, my name is ${writerName} ${writerSurname}. I am ${writerAge} years old.`)
 }
```

Comment:
Great , <here the same=> if you are using a loop and if state it would be much easier>